requirements-freqai.txt

```diff
@@ -10,3 +10,4 @@ catboost==1.2; 'arm' not in platform_machine and (sys_platform != 'darwin' or py
 lightgbm==3.3.5
 xgboost==1.7.5
 tensorboard==2.13.0
+datasieve==0.1.4
```

Comment:
should be added to setup.py as well.
if we require a specific min-version - best using `>=0.1.4` or similar (it'll usually not be a pinned version there - but should still be there to facilitate installs from pypi, for example `pip install freqtrade[freqai]`.

While not a documented install method (thanks ta-lib), it's still one that's used by some people.

Reply:
Done